packages/vow/src/tools.js

```diff
@@ -33,6 +34,28 @@ export const prepareVowTools = (zone, powers = {}) => {
   const watchUtils = makeWatchUtils();
   const asVow = makeAsVow(makeVowKit);
 
+  /**
+   * TODO FIXME make this real
+   * Create a function that retries the given function if the underlying
+   * functions rejects due to upgrade disconnection.
+   *
+   * The internal functions
+   *
+   * @template T
+   * @param {Zone} fnZone - the zone for the named function
+   * @param {string} name
+   * @param {(...args: unknown[]) => ERef<T>} fn
+   * @returns {(...args: unknown[]) => Vow<T>}
+   */
+  const retriable =
+    (fnZone, name, fn) =>
```

Comment:
Wondering if we should put a dummy `fnZone.prepareExo` here to force the user to call `retriable` top level, even though that would only let us detect issues during upgrades.